contrib/dracut/90zfs/module-setup.sh.in

```diff
@@ -58,7 +58,7 @@ install() {
 		inst_script "${moddir}/zfs-generator.sh" "$systemdutildir"/system-generators/dracut-zfs-generator
 	fi
 	inst_hook mount 98 "${moddir}/mount-zfs.sh"
-	inst_hook shutdown 30 "${moddir}/export-zfs.sh"
+	inst_hook shutdown 20 "${moddir}/export-zfs.sh"
```

Comment:
I am running it right now.  I consider it ready to be merged.  I am happy with the result.

You can independently verify with `zfs-fedora-installer` as well.